src/main/java/eu/europa/ec/dgc/gateway/restapi/mapper/CertificateRoleMapper.java

```diff
@@ -0,0 +1,39 @@
+/*-
+ * ---license-start
+ * EU Digital Green Certificate Gateway Service / dgc-gateway
+ * ---
+ * Copyright (C) 2021 T-Systems International GmbH and all other contributors
+ * ---
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ---license-end
+ */
+
+package eu.europa.ec.dgc.gateway.restapi.mapper;
+
+import eu.europa.ec.dgc.gateway.entity.TrustedPartyEntity;
+import eu.europa.ec.dgc.gateway.restapi.filter.CertificateAuthenticationRole;
+import org.mapstruct.Mapper;
+import org.mapstruct.ValueMapping;
+import org.mapstruct.ValueMappings;
+
+@Mapper(componentModel = "spring")
+public interface CertificateRoleMapper {
+
+    @ValueMappings({
```

Comment:
ValueMappings can be removed with java >= 8, see https://mapstruct.org/documentation/stable/api/org/mapstruct/ValueMappings.html

Reply:
Removed it